client/landing/stepper/declarative-flow/internals/components/migration-assistance-modal/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { useTranslate } from 'i18n-calypso';
+import ConfirmModal from 'calypso/blocks/importer/components/confirm-modal';
+import './style.scss';
+interface MigrationAssistanceModalProps {
+	navigateBack: ( () => void ) | undefined;
+	migrateFrom: string | null;
+	onConfirm: ( () => void ) | undefined;
+}
+export const MigrationAssistanceModal: React.FunctionComponent< MigrationAssistanceModalProps > = (
+	props: MigrationAssistanceModalProps
+) => {
+	const translate = useTranslate();
+	const importSiteHostName = props.migrateFrom || translate( 'your site' );
+
+	return (
+		<ConfirmModal
+			compact={ false }
+			title={ translate( 'Migration sounds daunting? It shouldn’t be!' ) }
+			confirmText={ translate( 'Take the deal' ) }
+			cancelText={ translate( 'No, thanks' ) }
+			onClose={ props.navigateBack }
+			onConfirm={ () => {} }
+		>
+			<p>
+				{ translate(
+					`Subscribe to the Creator plan now, and get a complimentary migration service (normally $500) to move %(importSiteHostName)s to WordPress.com.`,
```

Comment:
Let's add a 
```
/* translators:  .... */
```

Reply:
![image](https://github.com/Automattic/wp-calypso/assets/52076348/07e8a159-20ec-4e93-a98f-e9bf4e1d5ae5)

Let's be sure that Creator plan and the name of the website are in bold :)

Reply:
Since it won't be visible to users for now, I haven't put much effort on these since the designs are not 100% done and it might change. Same with the copies.

Reply:
Yeah that's ok, better to focus on the structural part. But let's do not forget about addressing this, as copy may remain this